lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+  attr_reader :meals
```

Comment:
This is one where I guess I went beyond the user stories and was thinking about how the API would look from the restaurant side too, so had made the menu able to be amended. If the code was going to be shared across restaurants, you also wouldn't want it to have a default @meals attribute. Obviously this did mean every time I tested my code in irb I had to create new menus, which was a bit long!

Reply:
I agree. I still like the idea of having a default menu stored as a constant - for ease of loading, and it can be initialized with different menus. But in hindsight I would add more flexibility by letting the menu be customisable by the restaurant through adding and removing dishes.